configurations/perf/regression-nosqlbench-gce.yaml

```diff
@@ -0,0 +1,8 @@
+test_duration: 360
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
```

Comment:
also in tet yaml, because it's common to all configurations.

Reply:
Fixed